test/EthernaIndex.Services.Tests/Tasks/MetadataVideoValidatorTaskTest.cs

```diff
@@ -428,7 +428,7 @@ public async Task ValidateManifest_AppendManifestInVideo_WhenIsValidAndHaveAnoth
             secondSwarmService
                 .Setup(x => x.GetMetadataVideoAsync(secondManifestHash))
                 .ReturnsAsync(secondMetadataVideoDto);
-            var secondMetadataVideoValidatorTask = new MetadataVideoValidatorTask(secondIndexContext.Object, secondSwarmService.Object);
+            IMetadataVideoValidatorTask secondMetadataVideoValidatorTask = new MetadataVideoValidatorTask(secondIndexContext.Object, secondSwarmService.Object);
```

Comment:
It's not required to replace var with interface. The point is to use dependency injection